jsk_robot_common/jsk_robot_startup/lifelog/common_logger.launch

```diff
@@ -310,6 +339,22 @@
       </rosparam>
     </node>
 
+    <!-- human pose logger -->
+    <node if="$(arg save_human_pose)"
+          name="human_pose_logger"
+          pkg="jsk_robot_startup" type="mongo_record.py"
+          machine="$(arg machine)"
+          respawn="$(arg respawn)">
+      <rosparam subst_value="true">
+        database: $(arg database)
+        topics:
+        - /edgetpu_human_pose_estimator/output/poses
+      </rosparam>
+      <rosparam subst_value="true" if="$(eval arg('collection') != '')">
+        collection: $(arg collection)
+      </rosparam>
+    </node>
```

Comment:
can you also add logger fot `edgetpu_object_detector/output` too?

Reply:
Thank you. I added.